src/api/exchange/ExchangeApiMock.ts

```diff
@@ -23,8 +23,8 @@ export class ExchangeApiMock extends DepositApiMock implements ExchangeApi {
     balanceStates: BalancesByUserAndToken,
     erc20Api: Erc20Api,
     registeredTokens: string[] = [],
-    maxTokens = 10000,
     ordersByUser: OrdersByUser = {},
+    maxTokens = 10000,
```

Comment:
Is it time to switch to object as params? :smile:

Reply:
Well, can't say that I thought about that. But I like the suggestion. Will do!